src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/grok/autocomplete.ts

```diff
@@ -56,9 +56,13 @@ export async function autocomplete(
 
   // GROK /
   const fieldSuggestions = (await callbacks?.getByType?.(ESQL_STRING_TYPES)) || [];
-  return fieldSuggestions.map((sug) => ({
-    ...sug,
-    text: `${sug.text} `,
-    command: TRIGGER_SUGGESTION_COMMAND,
-  }));
+  return fieldSuggestions.map((sug) => {
+    // if there is already a command, we don't want to override it
+    if (sug.command) return sug;
```

Comment:
what is it `sug ` :D  ?

<img width="618" height="143" alt="Image" src="https://github.com/user-attachments/assets/c2cae400-b4d8-496c-864d-33daee5bfded" />